src/test/config-starknet.sol

```diff
@@ -0,0 +1,160 @@
+// SPDX-FileCopyrightText: © 2022 Dai Foundation <www.daifoundation.org>
```

Comment:
Shouldn’t this file be imported from the spell base ?

Reply:
Not sure what you mean. This one imports the base and creates an instance on ln 95.
https://github.com/makerdao/spells-mainnet/pull/253/files#diff-3f2567b0ccb7a9159011ea10fbf05b2611c206fff903e97c43f41fed84ece8beR95

Reply:
I would suggest a different name for this file. A name that includes `.t` because it has tests, and doesn’t include `config` because it behaves very differently from [our current config](https://github.com/makerdao/spells-mainnet/blob/master/src/test/config.sol).

`src/Starknet.t.sol` ?

Reply:
This one contains both the config and the tests. As we add these additional things to the protocol it's probably going to be cleaner to separate these tests out into their own files, but I didn't want to create separate files for the config and the test.